extensions/arch/catalogd.adoc

```diff
@@ -11,11 +11,6 @@ include::snippets/technology-preview.adoc[]
 
 {olmv1-first} uses the catalogd component and its resources to manage Operator and extension catalogs.
 
-[IMPORTANT]
-====
-include::snippets/olmv1-known-issue-private-registries.adoc[]
```

Comment:
This is probably really unimportant given that it seems this snippet won't be included in any files anymore, but would be it worth going to the snippet and cleaning up the "included in..." list at the top?

Reply:
I'll be deleting the snippet in a follow up PR.

Reply:
But a good call out!